devtools/conda-envs/examples.yaml

```diff
@@ -0,0 +1,26 @@
+name: offtk-examples
```

Comment:
If possible, let's see if we can piggyback off of `rdkit.yaml` and `openeye.yaml`, since it's easy for yamls in this directory to fall out of date in a fail-deadly way, so it's good to have fewer of them.